packages/headless/src/features/facets/facet-api/request.ts

```diff
@@ -69,6 +69,9 @@ export interface Expandable {
 }
 
 export interface Type<T extends FacetType> {
+  /**
+   * The type of facet.
+   */
```

Comment:
This should not need to be documented because it is not exposed. It does not hurt to, but is there a benefit of not documenting it to know that "this is not exposed"?